NSS-Day-Cohort-64/rare-django-server-rare-django-server-init-to-win-it
# added tag_view with POST, PUT, DESTROY methods functioning and proper…

Added tag view --
to test:
git checkout lw-startingover (this is a bad name i know im sorry i will keep this in mind moving forward)

Open Postman
do GET request to http://localhost:8000/tags with auth token: 06787add96eb3264080015997720cb0877b40f90
You should see a list of tags. Add /1 at the end to get a single tag

Do a POST request to http://localhost:8000/tags
{
"label": "suspenseful"
}

Do a PUT request to http://localhost:8000/posts/(whatever tag you created)
{
"label": "suspense"
}
Do a get request to the same post to make sure it updates

Do a DELETE request to http://localhost:8000/posts/(whatever tag you created)
Do a get request to verify that the tag was deleted.


I also added the HTTP 201 CREATED status code to the post and comment and category create methods :)
